streams/src/test/java/org/apache/kafka/streams/StreamsConfigTest.java

```diff
@@ -1375,6 +1375,18 @@ public void shouldReturnDefaultClientSupplier() {
         assertTrue(supplier instanceof DefaultKafkaClientSupplier);
     }
 
+    @Test
+    public void shouldReturnDefaultRackAwareAssignmentConfig() {
+        final String strategy = streamsConfig.getString(StreamsConfig.RACK_AWARE_ASSIGNMENT_STRATEGY);
+        assertEquals(StreamsConfig.RACK_AWARE_ASSSIGNMENT_STRATEGY_NONE, strategy);
+    }
+
```

Comment:
I think you also need to test setting the three valid values. In the end they are part of the public API, and should be a test that verifies that they are accepted. 

The test should be like this:
https://github.com/apache/kafka/blob/474053d2973b8790e50ccfe1bb0699694b0de1c7/streams/src/test/java/org/apache/kafka/streams/StreamsConfigTest.java#L615

Do not use constants `RACK_AWARE_ASSSIGNMENT_STRATEGY_NONE, RACK_AWARE_ASSSIGNMENT_STRATEGY_MIN_TRAFFIC, or RACK_AWARE_ASSSIGNMENT_STRATEGY_MIN_TRAFFIC_BALANCE_SUBTOPOLOGY` because you want to ensure that the test fails if the content of those constants changes.

Reply:
Make sense. I can use raw values